pages/team/[year].tsx

```diff
@@ -50,23 +56,18 @@ const Team = ({ year, teamMembers }: { year: string, teamMembers: Array<Member>
               </h5>
 
               <div className="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-5 lg:gap-2 items-center">
-                {teamMembers.map(
-                  (
-                    member,
-                    index
-                  ) => {
-                    if (member.domain == `${domain.name}`) {
-                      return (
-                        <MemberCard
-                          key={index}
-                          src={member.url}
-                          name={member.name}
-                          designation={member.designation}
-                        />
-                      )
-                    }
+                {teamMembers.map((member, index) => {
+                  if (member.domain == `${domain.name}`) {
```

Comment:
template string is not necessary here
```suggestion
                  if (member.domain === domain.name) {
```

Reply:
I dont remember doing this change, oh formatting